cpp/src/parquet/size_statistics_test.cc

```diff
@@ -376,4 +376,22 @@ TEST_F(SizeStatisticsRoundTripTest, LargePage) {
                                                         /*byte_array_bytes=*/{90000}}));
 }
 
+TEST_F(SizeStatisticsRoundTripTest, MaxLevelZero) {
```

Comment:
Should we add a file to parquet-testing that has max_def==1 and omits the def histogram?

Reply:
I don't think if any open-sourced Parquet writer is able to create this yet...

Reply:
I could coerce one into doing so. But with the change allowing empty histograms, it's probably not worth creating one now.